llvm/test/CodeGen/X86/statepoint-fixup-undef.mir

```diff
@@ -182,13 +181,13 @@ body:             |
   ; STACKMAP-NEXT:	.short	3
   ; STACKMAP-NEXT:	.short	0
   ; STACKMAP-NEXT:	.long	0
-  ;      This is entry we're looking for, reference to constant pool entry 0xFEFEFEFE
-  ; STACKMAP-NEXT:	.byte	5
+  ;      This is a constant 0xFEFEFEFE we are looking for
```

Comment:
`0xFEFEFEFE` is no longer sign-extended to int64_t and as such doesn't require a constant pool entry. Since, as I understand, this was added mostly to not crash and it doesn't havy specific meaning, I think we are fine with this change.

@arsenm I saw that you had a stack that removed this special handling completely, although it didn't go anywhere, even though some patches were approved https://reviews.llvm.org/D122605